python/pyspark/sql/streaming.py

```diff
@@ -786,9 +798,8 @@ def queryName(self, queryName):
         self._jwrite = self._jwrite.queryName(queryName)
         return self
 
-    @keyword_only
     @since(2.0)
-    def trigger(self, processingTime=None):
+    def trigger(self, trigger=None, processingTime=None):
```

Comment:
Added this before processingTime so that we can use positional param and  write `trigger(OneTime())` 
Does not break existing APIs.